torch/_meta_registrations.py

```diff
@@ -2387,9 +2388,20 @@ def upsample_nearest3d(input, output_size, scales_d=None, scales_h=None, scales_
     )
 
 
-@register_meta([aten.sort.default, aten.sort.stable])
-def meta_sort(self, stable=None, dim=-1, descending=False):
-    return torch.empty_like(self), torch.empty_like(self, dtype=torch.int64)
+@register_meta(
+    [aten.sort.default, aten.sort.stable, aten.sort.values, aten.sort.values_stable]
+)
+def meta_sort(self, stable=None, dim=-1, descending=False, values=None, indices=None):
```

Comment:
Cannot use `out_wrapper` here because of multiple overloads, so I write to out manually as in the `mean` ref.

Also, the signature here is not strictly correct (note the `*`), but it's probably fine(?).

```
- func: sort.values(       Tensor self,                  int dim=-1, bool descending=False, *, Tensor(a!) values, Tensor(b!) indices) -> (Tensor(a!) values, Tensor(b!) indices)
- func: sort.values_stable(Tensor self, *, bool? stable, int dim=-1, bool descending=False, Tensor(a!) values, Tensor(b!) indices) -> (Tensor(a!) values, Tensor(b!) indices)
- func: sort(              Tensor self,                  int dim=-1, bool descending=False) -> (Tensor values, Tensor indices)
- func: sort.stable(       Tensor self, *, bool? stable, int dim=-1, bool descending=False) -> (Tensor values, Tensor indices)
```